src/index.js

```diff
@@ -1,93 +1,68 @@
-const fs = require("fs"),
-    http = require("http"),
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
+import '@babel/polyfill';
```

Comment:
This dependency is not needed as of this time. Could you explain why you are importing it?

Reply:
@babel/polyfill is needed at the entry file so that es7 async and wait can be transpired and used in the codebase.